README.md

```diff
@@ -1 +1,153 @@
-# TDD
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 TDD <a name="about-project"></a>
+
+[TDD] Is an app making created making use of TDD methodology. It solves some mathematical problems name; Fizzbuzz, reverse, and Factorial.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Key Features
+
+- Write unit tests using Rspec by doing test-driven development (TDD).
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Example command:
+
+```sh
+1.use a browser
+2.use cable internet
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+
+  git clone https://github.com/deniodev/TDD.git
+  cd TDD
+
+
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+- Open the terminal to play with the commands.
+```
+
+### Run tests
+
+To run tests for the linter, run the following command:
+
+Example command:
+
+```sh
+rubocop -A
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ragan Hailu**
+
+- GitHub: [ragan](https://github.com/ragangithub)
+- LinkedIn: [Ragan Hailu](https://linkedin.com/in/raganhailu)
+
+👤 **Denio Nhanale**
+
+- GitHub: [ragan](https://github.com/deniodev)
+- LinkedIn: [Ragan Hailu](https://linkedin.com/in/denionhanale)
+
```

Comment:
![image](https://github.com/deniodev/TDD/assets/93039359/5be01480-1425-4bae-a4c9-d5fe071980fb)

**[OPTIONAL]**

- [ ] It would be nice if you correct the names here because I don't think you can also collaborate on your project with you 😄